spec/unit/lib/temporal/client/retryer.rb

```diff
@@ -0,0 +1,45 @@
+require 'temporal/client/retryer'
+require 'temporal/metadata'
+require 'time'
+
+describe Temporal::Client::Retryer do
+  it 'backs off and stops retrying eventually' do
+    timestamps = []
+    max_wait = 5.0
```

Comment:
Will this make the test take up to 5 seconds? That seems kind of long considering how quickly these run today